google/cloud/forseti/services/cli.py

```diff
@@ -334,6 +342,14 @@ def define_notifier_parser(parent):
               'will be used.')
     )
 
+    create_notifier_parser.add_argument(
+        '--scanner_index_id',
+        default=0,
```

Comment:
What does a default of `0` do?  Would a `None` be more appropriate?

Reply:
agree, however the previous test case, which is for inventory_index_id, defaults to 0. Hmm would it be nice to be consistent?